### PR TITLE
Small fixes for Ubuntu 20.04

### PR DESCRIPTION
**Pull request by pieleric**

### CODE REVIEW

**Review by VictoriaDel — APPROVED** [review]

File: src/odemis/gui/main.xrc [hunk 1]
```diff
@@ -2978,7 +2976,7 @@
                       </XRCED>
                     </object>
                     <option>1</option>
-                    <flag>wxALL|wxEXPAND|wxALIGN_BOTTOM|wxALIGN_CENTRE_VERTICAL</flag>
+                    <flag>wxALL|wxEXPAND</flag>
```
Comment (tepals) [inline src/odemis/gui/main.xrc:2979]: Why is the align bottom also removed here?

File: src/odemis/gui/main_xrc.py [hunk 2]
```diff
@@ -3917,7 +3915,7 @@ def __init_resources():
                       </XRCED>
                     </object>
                     <option>1</option>
-                    <flag>wxALL|wxEXPAND|wxALIGN_BOTTOM|wxALIGN_CENTRE_VERTICAL</flag>
```
Comment (tepals) [inline src/odemis/gui/main_xrc.py:3920]: Why is the align bottom also removed here?

File: src/odemis/gui/plugin/__init__.py [hunk 3]
```diff
@@ -409,7 +409,7 @@ def addButton(self, label, callback=None, face_colour='def'):
                               style=wx.ALIGN_CENTER, face_colour=face_colour)
         self.buttons.append(btn)
         sizer = self.pnl_buttons.GetSizer()
-        sizer.Add(btn, proportion=1, flag=wx.ALL | wx.ALIGN_RIGHT, border=10)
+        sizer.Add(btn, proportion=1, flag=wx.ALL, border=10)
```
Comment (tepals) [inline src/odemis/gui/plugin/__init__.py:412]: why is align_right removed here?

File: src/odemis/gui/main_xrc.py (hunk 2 above)
Comment (pieleric) [reply src/odemis/gui/main_xrc.py:3920]: All ALIGN_* are incompatible with EXPAND.

**Review by tepals — APPROVED** [review]

File: src/odemis/gui/plugin/__init__.py (hunk 3 above)
Comment (pieleric) [reply src/odemis/gui/plugin/__init__.py:412]: Good question. I've checked. It's another rule, that I forgot to indicate in the commit log: as the sizer is horizontal, it's not possible to specify horizontal alignment. I've updated the commit log.

**Pull request merged**